x-pack/test/alerting_api_integration/spaces_only/tests/alerting/ml_rule_types/anomaly_detection/alert.ts

```diff
@@ -0,0 +1,295 @@
+      await ml.api.createDatafeed(getDatafeedConfig(), Spaces.space1.id);
+      await ml.api.openAnomalyDetectionJob(AD_JOB_ID);
+      await ml.api.startDatafeed(DATAFEED_ID);
+    });
+
+    afterEach(async () => {
+      await objectRemover.removeAll();
+      await esTestIndexTool.destroy();
+      await esTestIndexToolOutput.destroy();
+      await ml.api.deleteAnomalyDetectionJobES(AD_JOB_ID);
+      await ml.api.cleanMlIndices();
+      await ml.api.deleteIndices(BASIC_TEST_DATA_INDEX);
+    });
+
+    it('runs correctly', async () => {
+      await createAlert({
+        name: 'Test AD job',
+        // To make sure the alert is triggered ASAP
+        includeInterim: true,
+        jobSelection: {
+          jobIds: [AD_JOB_ID],
+        },
+        severity: 0,
+        lookbackInterval: undefined,
+        resultType: 'bucket',
+        topNBuckets: 3,
+      });
+
+      //  Ingest anomalous records
+      await ingestAnomalousDoc(BASIC_TEST_DATA_INDEX);
+
+      log.debug('Wait for bucket to finalize...');
+      await sleep(60000);
+
+      log.debug('Checking created alert instances...');
+
+      const docs = await waitForDocs(1);
+      for (const doc of docs) {
+        const { name, message } = doc._source.params;
+
+        expect(name).to.be('Test AD job');
+        expect(message).to.be(
+          'Alerts are raised based on real-time scores. Remember that scores may be adjusted over time as data continues to be analyzed.'
+        );
+      }
+    });
+
+    async function waitForDocs(count: number): Promise<any[]> {
+      return await esTestIndexToolOutput.waitForDocs(
+        ES_TEST_INDEX_SOURCE,
+        ES_TEST_INDEX_REFERENCE,
+        count
+      );
+    }
+
+    async function createAlert({
+      name,
+      ...params
+    }: MlAnomalyDetectionAlertParams & { name: string }): Promise<string> {
+      log.debug(`Creating an alerting rule "${name}"...`);
+      const action = {
+        id: actionId,
+        group: ANOMALY_SCORE_MATCH_GROUP_ID,
+        params: {
+          documents: [
+            {
+              source: ES_TEST_INDEX_SOURCE,
+              reference: ES_TEST_INDEX_REFERENCE,
+              params: {
+                name: '{{{alertName}}}',
+                message: '{{{context.message}}}',
+              },
+            },
+          ],
+        },
+      };
+
+      const { status, body: createdAlert } = await supertest
+        .post(`${getUrlPrefix(Spaces.space1.id)}/api/alerting/rule`)
+        .set('kbn-xsrf', 'foo')
+        .send({
+          name,
+          consumer: 'alerts',
+          enabled: true,
+          rule_type_id: ALERT_TYPE_ID,
+          schedule: { interval: `${ALERT_INTERVAL_SECONDS}s` },
+          actions: [action],
+          notify_when: 'onActiveAlert',
+          params,
+        });
+
+      // will print the error body, if an error occurred
+      if (status !== 200) {
+        // console.log(createdAlert);
+      }
```

Comment:
If you want to keep the logging during test execution here, you could use the `log` service. Otherwise, we should probably remove this part and go back to `.expect(200)`.

Reply:
Removed in https://github.com/elastic/kibana/pull/118726/commits/e9d1af78c048177a3c355ca325c8f3037d0d8c72